storage_backends.py

```diff
@@ -5,23 +5,28 @@
     import dropbox
 except ImportError:
     pass
-
+try:
+    import webdav3.client as wd_client
+    from io import BytesIO
```

Comment:
You can always import this at the beginning of this file as it's a native module (even if it's not always used).

Reply:
Addressed in latest commits.